lib/logical/ElementDefinition.js

```diff
@@ -640,6 +640,37 @@ class ElementDefinition {
     return this;
   }
 
+  /**
+   * Fixes a fixed value to the element or a child of the element indicated by the path.  If the target element is
+   * in a deeper path than currently represented in the structure definition, parent level elements will be "expanded"
+   * (or "unrolled") based on their definition.  This means that calling this function may result in increasing the
+   * number of elements in the structure definition.
+   * @param {boolean} value - the fixed value to fix
+   * @param {string} [path] - if the fixed should be fixed to a child element, the dot-separated path of that element
+   * @param {function({code: string, profile?: string[], targetProfile?: string[], aggregation?: string[], versioning?: string}):StructureDefinition} [resolve] - a function that can resolve a type to a StructureDefinition instance
+   * @returns {ElementDefinition} the element to which the value was fixed (may be a child of the current element)
+   */
+  fixFixedValue(value, path, resolve) {
```

Comment:
This seems redundant.  Why not just `fixValue`?

Reply:
Fixed here https://github.com/standardhealth/shr-fhir-export/pull/163/commits/f2284c8340cf9eae02f3555806b6255e3d825a50